_episodes/01-starting-with-data.md

```diff
@@ -208,22 +208,29 @@ We'll talk a bit more about what the different formats mean in a different lesso
 
 ### Useful Ways to View DataFrame objects in Python
 
-There are multiple methods that can be used to summarize and access the data
-stored in DataFrames. Let's try out a few. Note that we call the method by using
-the object name `surveys_df.method`. So `surveys_df.columns` provides an index
-of all of the column names in our DataFrame.
+There are many ways to summarize and access the data stored in DataFrames, 
+using attributes and methods provided by the DataFrame object. Attributes 
+are called using the DataFrame object name `surveys_df.attribute`, so the 
+attribute `surveys_df.columns` provides an index of all of the column names 
+in our DataFrame.
 
-## Challenges
+Methods are called using the syntax `surveys_df.method()` and we can supply extra 
```

Comment:
Perhaps:

Methods are called in a similar fashion using the syntax `df_object.method()`. As an example, `survey_df.head()` gets the first few rows in a DataFrame `survey_df` using the `head()` method. With a method, we can supply extra information in the parens to control behaviour.